### ISSUE TITLE
[Enhancement] Add the option to generate secondary albedo and normal, to account for reflections.

### ISSUE THREAD

**Issue by ProNoob135** [leMaik/chunky-denoiser#9]
Add a second normal and albedo, (both off by default?) that calculates that applies to reflected surfaces. Maybe also add the option to combine them with the first based on how reflective the surface is. Will make water look much better.

Also maybe a third for refracted albedo/normal, which will fix underwater areas getting blurred. (also with the option to automatically combine with the first/second.)

**Comment by leMaik** [comment]
Good idea! How can multiple albedo maps be used with Intel Open Image Denoise? Do you have some documentation on how exactly these additional albedo and normal maps should look?

**Comment by ProNoob135** [comment]
It doesn't have multiple inputs, but by blending the normal/albedo based on the reflectivity of the surface it should work in theory, Or you can denoise them separately and then cut out parts of the image and blend them in post.

**Comment by ProNoob135** [comment]
This would probably be a better discussion for discord

**Comment by jackjt8** [comment]
![image](https://user-images.githubusercontent.com/3740917/80248283-86d3c380-8667-11ea-9e39-c81d2913f255.png)

Oinky-McWoof made a good render showing some of the current issues.
